Dockerfile

```diff
@@ -61,26 +63,21 @@ RUN make -f docker.make build_deps
 COPY ./requirements/*.txt /app/requirements/
 RUN make -f docker.make build_python_deps
 
-# Install the application code.
+# Install geocalc.
 COPY . /app
-RUN make -f docker.make build_ichnaea
+RUN make -f docker.make build_geocalc
```

Comment:
Previously, this would build and install ichnaea which included building and installing the geocalc extension. I moved the geocalc code into a separate Python package, so that package gets built and installed, but the ichnaea package no longer does.

That fixes the issue where you had to install ichnaea in order to see changes you're making. It also fixes the issue where building geocalc creates build artifacts in `/app` which then get shadowed when the repository directory is mounted as `/app`.